### DISCUSSION TITLE
When will the next version of Pydantic be released?

### DISCUSSION THREAD

**Discussion by JohnVillalovos** [pydantic/pydantic#3503] [Feature Request]
So I was wondering when will a new release be made of Pydantic?

Any plans on setting up an automated release system to make a new release every X unit_of_time?

Thanks!

Some data points I came up with:

- Current version of  pydantic on PyPI is 1.8.2
- Was released: on May 11, 2021 which was over 200 days ago as of today.
- There appear to be over 140 commits done to `master` since the `v1.8.1` tag and `v1.8.1-branch` branch.
- The `v1.8.2` tag looks like it diverged from `master` after `v1.8.1` with about 16 commits.

**Comment by NowanIlfideme** [comment]
FYI 1.9 has been released. And 1.10 might exist, might not - I believe SamuelColvin is considering features of 2.X currently.
